pom.xml

```diff
@@ -11,6 +11,12 @@
     <!--module>test</module-->
   </modules>
 
+  <properties>
+    <java.level>6</java.level>
```

Comment:
This change requires java6 as `PropertyResourceBundle(Reader)` is available since Java6: https://docs.oracle.com/javase/7/docs/api/java/util/PropertyResourceBundle.html#PropertyResourceBundle(java.io.Reader)